src/sentry/static/sentry/app/components/smartSearchBar/index.jsx

```diff
@@ -19,7 +19,7 @@ import {
 } from 'app/actionCreators/savedSearches';
 import {t} from 'app/locale';
 import Button from 'app/components/button';
-import CreateSavedSearchButton from 'app/views/stream/createSavedSearchButton';
```

Comment:
I wonder if we even need this `organization` designation at all for our upper level views since project settings are really the only things that should be project specific now.

Reply:
Yeah, it does seem a lot less meaningful now that everything is organization level. I can drop the organization prefix if we agree that's the naming direction we want to go.

Reply:
we should decide + do this in another PR

Reply:
I'm :+1: on removing the prefix. It doesn't add anything when we only have organization scoped views.

Reply:
Would it be worthwhile to discuss this + the root `views` directory in general tomorrow in the TSC?

Reply:
I think so.